src/MeetingsSDKAdapter.js

```diff
@@ -194,6 +211,116 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
     });
   }
 
+  /**
+   * Attempts to mute the microphone of the given meeting ID.
+   * If the microphone is successfully muted, an audio mute event is dispatched.
+   *
+   * @param {string} ID ID of the meeting to mute audio
+   * @memberof MeetingsSDKAdapter
+   */
+  muteAudioMeeting(ID) {
+    const sdkMeeting = this.fetchMeeting(ID);
+
+    sdkMeeting
+      .muteAudio()
+      .then(() => {
+        sdkMeeting.emit(EVENT_MEDIA_LOCAL_MUTE, {type: MEDIA_TYPE_LOCAL_MUTE_AUDIO});
```

Comment:
Awesome solution for utilizing the sdk's `emit` functionality!

Reply:
Could you leave a comment that says why we are creating this event due to SDK limitations around promises?